### DISCUSSION TITLE
Access token not stored in the environment in latest version

### DISCUSSION THREAD

**Discussion by Greentears** [OpenIDC/mod_auth_openidc#1289] [Q&A] [answered]
We're using mod_auth_openidc 2.4.15.3 successfully and it correctly transmits the access token to the environment.
We tested version 2.4.16.5 and, with the exact same configuration as before, the access token is not stored in the environment anymore.
We tried to debug the code, but we couldn't understand why it doesn't work.

We might think there’s an issue, do you have any idea why we're having that kind of problem ?

**Comment by zandbelt** [comment, marked as answer]
the server logs (`LogLevel` set do `debug`) would tell you what is going on
